Various Fixes

SgtBears opened the pull request:
You need to compile the dll for this to work. This changes a few more tags, makes more tags optional, adds translations for Dynamic Names and Fixes a few text problems with the new buildings.

Included fix for URV text malfunction.

SgtBears committed:
Merge

SgtBears committed:
URV Text Fix

dguenms commented on Assets/XML/Civilizations/CIV4CivilizationInfos.xml:
```diff
@@ -3017,15 +3017,15 @@
 			<ArtStyleType>ARTSTYLE_AFRICAN</ArtStyleType>
-			<UnitArtStyleType>UNIT_ARTSTYLE_MIDDLE_EAST</UnitArtStyleType>
+			<UnitArtStyleType>UNIT_ARTSTYLE_MIDDLE_EASTERN</UnitArtStyleType>
 			<Identifier>KON</Identifier>
```
Shouldn't the identifier also change?

dguenms commented on Assets/XML/Civilizations/CIV4UnitArtStyleTypeInfos.xml:
```diff
@@ -20,17 +20,22 @@
-						<EarlyArtDefineTag>ART_DEF_UNIT_ASIAN_WAR_GALLEY</EarlyArtDefineTag>
-						<MiddleArtDefineTag>ART_DEF_UNIT_ASIAN_WAR_GALLEY</MiddleArtDefineTag>
-						<LateArtDefineTag>ART_DEF_UNIT_ASIAN_WAR_GALLEY</LateArtDefineTag>
+						<EarlyArtDefineTag>ART_DEF_UNIT_ASIAN_ASIAN</EarlyArtDefineTag>
```
That looks like a mistake?

SgtBears commented:
I wasn't sure how the identifier was used so I didn't want to mess with it. If you tell me where it is used I can change it.

Fixed the Asian caravel.

SgtBears committed:
Fixes for Asian Caravel

dguenms commented on Assets/XML/Text/DynamicNames/DynamicNames.xml:
```diff
@@ -40,6 +40,26 @@
+			<Plural>0</Plural>
+		</French>
+		<German>
+			<Text>K&#246;nigreich %s1:K&#246;nigreich %s1:K&#246;nigreich %s1</Text>
```
You don't have to escape unicode characters in XML, please just write them out in plain text to make the files more legible.

SgtBears commented:
Fixed the Identifier.

Fixed the Asian caravel.

I just copied the translations from the original RFC by comparing the english. I don't know what the characters in the translations are supposed to be. I had a quick look but there doesn't look like there is a quick way of changed the unicode characters to normal characters.

SgtBears committed:
Fixed the Identifier.

dguenms commented:
Can you take out all files where Github cannot display the diff and put them in a separate pull request? I'd like to see what exactly has changed.

SgtBears commented:
I haven't removed the files from this merge but I added the files to a different merge so you can view them.

dguenms commented:
Okay, nothing to comment on for those files.

Otherwise things would have become really confusing though. You're making it really hard for me by submitting those huge pull requests. It's much easier to separate them by purpose. That you have to put various in the title should be the first warning sign.

**At the very least** make sure that all file diffs are viewable on Github. But I'd prefer more separate pull requests because then I can merge them incrementally depending on their status instead of this all or nothing deal.

This pull request also changes a lot of unit tags. Have you made sure that Varietas Delectat still works with this? I'll hold off on this until that is the case.

SgtBears commented:
Okay, I'll try to create smaller pull requests.

The updated varietas Delectat that I haven't uploaded yet is compatible. Normal varietas delectat isn't.

SgtBears committed:
Jewish Missionary Bug

SgtBears commented:
I uploaded the updated varietas delectat however the changelog is impossible to read.

dguenms commented:
Okay, I'm waiting for the pull request against VD then.

SgtBears commented:
Its in the Varietas delectat section.

SgtBears committed:
Merge branch 'develop' of https://github.com/dguenms/Dawn-of-Civilization into Other2

SgtBears committed:
Fixes

SgtBears committed:
Leader Fixes

SgtBears committed:
Vargas Fix

SgtBears committed:
Merge branch 'develop' of https://github.com/dguenms/Dawn-of-Civilization into Other2

# Conflicts:
#	Assets/XML/Text/DynamicNames/DynamicNames_Persia.xml

SgtBears committed:
Merge branch 'develop' of https://github.com/dguenms/Dawn-of-Civilization into Other2

# Conflicts:
#	Assets/Python/Consts.py
#	Assets/Python/GreatPeople.py

SgtBears commented:
Updated and have added the new leaders civilopedia texts.

SgtBears committed:
Fixes

SgtBears committed:
Fixes
